media/org.openttd.OpenTTD.metainfo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--Copyright 2018-2020 Flathub maintainers-->
```

Comment:
They cannot be the copyright holder if it is included in our source-tree. So you will have to check with them if they are willing to transfer the copyright to us.

Reply:
The file is under CC0-1.0 license, so this should be just about having authors specified. But I have created issue for that, so they can react any way.

Reply:
Copyright and license are two different things :) They can license it under CC0, but they still hold the copyright :) A bit silly for a file like this, ofc, but they added the copyright, not me :P

Reply:
> They can license it under CC0, but they still hold the copyright

"The person who associated a work with this deed has dedicated the work to the public domain by **waiving all of his or her rights to the work worldwide under copyright law**, including all related and neighboring rights, to the extent allowed by law. "

Reply:
> A bit silly for a file like this

Your judgement call put aside, the copyright header was more of an authorship statement than anything. Copyright is waived as per the quote of the CC0 above. Feel free to remove it.